node/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM kindest/node:v1.21.12
+FROM kindest/node:v1.24.0
```

Comment:
I don't think this will work since we'd have to switch to `systemd` cgroup driver in gardener/gardener (ref https://github.com/gardener/gardener/issues/5325).